meson.build

```
@@ -1,18 +1,38 @@
-project('xmpp-vala', 'vala')
+project('xmpp-vala', 'c', 'cpp', 'vala')
 
 fs = import('fs')
+gnome = import('gnome')
+i18n = import('i18n')
 python = import('python')
 
 dep_gdk_pixbuf = dependency('gdk-pixbuf-2.0')
 dep_gee = dependency('gee-0.8')
 dep_gio = dependency('gio-2.0')
 dep_glib = dependency('glib-2.0')
+dep_gnutls = dependency('gnutls')
 dep_gmodule = dependency('gmodule-2.0')
+dep_gpgme = dependency('gpgme')
+dep_gstreamer = dependency('gstreamer-1.0')
+dep_gstreamer_app = dependency('gstreamer-app-1.0')
+dep_gstreamer_audio = dependency('gstreamer-audio-1.0')
+dep_gstreamer_rtp = dependency('gstreamer-rtp-1.0')
+dep_gstreamer_video = dependency('gstreamer-video-1.0')
 dep_gtk4 = dependency('gtk4')
 dep_icu_uc = dependency('icu-uc')
 dep_libadwaita = dependency('libadwaita-1')
+dep_libcanberra = dependency('libcanberra')
+dep_libgcrypt = dependency('libgcrypt')
+dep_libqrencode = dependency('libqrencode')
+dep_libsrtp2 = dependency('libsrtp2')
+# libsignal-protocol-c has a history of breaking compatibility on the patch level
+# we'll have to check compatibility for every new release
+# distro maintainers may update this dependency after compatibility tests
+dep_libsignal_protocol_c = dependency('libsignal-protocol-c', version: '>=2.3.2,<2.3.4')
```

Probably this should be an array of versions:

```suggestion
dep_libsignal_protocol_c = dependency('libsignal-protocol-c', version: ['>=2.3.2'], ['<2.3.4'])
```

---

Thanks! Done.